### PR TITLE
Clutter Project for win32

### PR DESCRIPTION
**Pull request by hsccr**
Hello! 

I am Korean guy.

I have a poor English. so you can understand me.

I received mail from you but I can't reply. sorry!

I am porting open-gl based Linux program to win32 program.

I received a lot of help from wingtk.

so, I want to pull request some project for currently working.

please reviews and confirm.

thanks.

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -250,6 +250,33 @@ def build(self):
...
+    def __init__(self):
+        Project.__init__(self,
+            'clutter',
+            archive_url = 'http://ftp.acc.umu.se/pub/GNOME/sources/clutter/1.26/clutter-1.26.0.tar.xz',
+            dependencies = ['atk','cogl','json-glib'],
+            )
+
+    def build(self):
+        self.exec_msbuild(r'build\win32\vs%(vs_ver)s\clutter.sln')
+
```
Comment (nacho) [inline build.py]: you should install the license

File: build.py [hunk 2]
```diff
@@ -250,6 +250,33 @@ def build(self):
...
+        Project.__init__(self,
+            'cogl',
+            archive_url = 'http://ftp.acc.umu.se/pub/GNOME/sources/cogl/1.22/cogl-1.22.0.tar.xz',
+            dependencies = ['glib','cairo','pango','gdk-pixbuf'],
+            patches = ['001-cogl-missing-symbols.patch','002-cogl-pango-missing-symbols.patch','003-cogl-framebuffer-missing-debug-return.patch'],
+            )
+
+    def build(self):
+        self.exec_msbuild(r'build\win32\vs%(vs_ver)s\cogl.sln')
+
```
Comment (nacho) [inline build.py]: you have to install the license

**Comment by hsccr** [comment]
I am the windows programmer.

so I don't know about Linux based open source license.

could you explain how to install the license?


I

**Comment by nacho** [comment]
See for example other projects in build.py:
https://github.com/wingtk/gtk-win32/blob/master/build.py#L416

basically we do a self.install('COPYING share/doc/name'), you should install the license from cogl and from clutter.

**Comment by hsccr** [comment]
fixed to install the license.

**Pull request closed without merging**